apps/web/src/modules/job/list/pages/types.ts

```diff
@@ -47,6 +49,8 @@ export const initialIFilter: IFilter = {
   maleCheck: false,
   femaleCheck: false,
   otherCheck: false,
+  cancleCheck: false,
```

Comment:
Please change all object key and types which related to this also
```suggestion
  cancelCheck: false,
```